tethys_cli/cli_helpers.py

```diff
@@ -80,6 +80,31 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
+def prompt_yes_or_no(question, default="y"):
```

Comment:
I'm surprised we don't have anything like this anywhere already. Did you do some looking in the other CLI commands?